themes/lucky/lucky.theme.sh

```diff
@@ -0,0 +1,59 @@
+#! bash oh-my-bash.module
+
+_omb_deprecate_const 20000 red "$_omb_prompt_brown" "${_omb_deprecate_msg_please_use/'%s'/_omb_prompt_brown}"
+_omb_deprecate_const 20000 blue "$_omb_prompt_navy" "${_omb_deprecate_msg_please_use/'%s'/_omb_prompt_navy}"
+_omb_deprecate_const 20000 green "$_omb_prompt_green" "${_omb_deprecate_msg_please_use/'%s'/_omb_prompt_green}"
+_omb_deprecate_const 20000 purple "$_omb_prompt_purple" "${_omb_deprecate_msg_please_use/'%s'/_omb_prompt_purple}"
```

Comment:
Could you explain these lines? These define variables `red`, `blue`, `green`, and `purple`, but none of them seem to be used in the theme. In the first place, these variables are prepared by `lib/omb-deprecated.sh`, which is always loaded. So these variables already exist.